addons/udes_stock_cron/models/stock_move_line.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+from odoo import models, fields, _
+from collections import defaultdict
+
+class StockMoveLine(models.Model):
+    _inherit = "stock.move.line"
+
+    def any_destination_locations_default(self):
+        """Checks if all location_dest_id's are the picks default
```

Comment:
nit: Checks if any not all.
(Do we want this here or in udes_stock -seems pretty generic?)